### ISSUE TITLE
stylish-haskell smashes safe imports

### ISSUE THREAD

**Issue by konn** [jaspervdj/stylish-haskell#90]
If we apply `stylish-haskell` to the following code, `safe` import is gone...

before:

``` haskell
{-# LANGUAGE SafeImports #-}
module Main where
import safe Prelude

main :: IO ()
main = undefined
```

after: 

``` haskell
{-# LANGUAGE SafeImports #-}
module Main where
import Prelude

main :: IO ()
main = undefined
```

**Comment by bitemyapp** [comment]
It does to `import {-# SOURCE #-} Module` imports as well.

**Comment by danplubell** [comment]
How should the following examples format for each Align value?
import qualified MyQualifiedModule QM
import MySimpleModule
import {-# SOURCE #-} MyModule1
import {-# SOURCE #-} qualified MyModule2 as MM
import safe MyModule3
import safe qualified MyModule4 as M4
import "base" Data.Foldable (foldMap)
import qualified "containers" Data.Map as Map

**Comment by alexkazik** [comment]
It's better to just skip them from being formatted instead of dropping the information like it's now.

**Comment by neongreen** [comment]
It has been fixed in #143.
